var/spack/repos/builtin/packages/llvm-flang/package.py

```diff
@@ -0,0 +1,169 @@
+             placement='clang',
+             when='@20190329')
+
+    resource(name='flang-driver',
+             git='https://github.com/flang-compiler/flang-driver.git',
+             tag='flang_20181226_70',
+             destination='tools',
+             placement='clang',
+             when='@20181226_70')
+
+    resource(name='flang-driver',
+             git='https://github.com/flang-compiler/flang-driver.git',
+             tag='flang_20181226_60',
+             destination='tools',
+             placement='clang',
+             when='@20181226_60')
+
+    resource(name='flang-driver',
+             git='https://github.com/flang-compiler/flang-driver.git',
+             tag='flang_20180921',
+             destination='tools',
+             placement='clang',
+             when='@20180921')
+
+    resource(name='flang-driver',
+             git='https://github.com/flang-compiler/flang-driver.git',
+             tag='flang_20180921',
+             destination='tools',
+             placement='clang',
+             when='@20180308')
+
+    resource(name='openmp',
+             git='https://github.com/flang-compiler/openmp.git',
+             branch='master',
+             destination='projects',
+             placement='openmp',
+             when='@master')
+
+    resource(name='openmp',
+             git='https://github.com/flang-compiler/openmp.git',
+             branch='release_70',
+             destination='projects',
+             placement='openmp',
+             when='@release_70')
+
+    resource(name='openmp',
+             git='https://github.com/flang-compiler/openmp.git',
+             branch='release_60',
+             destination='projects',
+             placement='openmp',
+             when='@release_60')
+
+    resource(name='openmp',
+             git='https://github.com/flang-compiler/openmp.git',
+             tag='flang_20190329',
+             destination='projects',
+             placement='openmp',
+             when='@20190329')
+
+    resource(name='openmp',
+             git='https://github.com/flang-compiler/openmp.git',
+             tag='flang_20181226_70',
+             destination='projects',
+             placement='openmp',
+             when='@20181226_70')
+
+    resource(name='openmp',
+             git='https://github.com/flang-compiler/openmp.git',
+             tag='flang_20181226_60',
+             destination='projects',
+             placement='openmp',
+             when='@20181226_60')
+
+    resource(name='openmp',
+             git='https://github.com/flang-compiler/openmp.git',
+             tag='flang_20180921',
+             destination='projects',
+             placement='openmp',
+             when='@20180921')
+
+    def cmake_args(self):
+        spec = self.spec
+        args = []
+        args.append('-DPYTHON_EXECUTABLE={0}'.format(
+            spec['python'].command.path))
+
+        if '+all_targets' not in spec:  # all is default on cmake
+            targets = []
```

Comment:
Why use a list? It seems like there will only ever be one possible value with your code.

Reply:
I think you are right. I was following the example in the main `llvm` package, but that package has support for GPUs and other stuff which is why they need the list. will fix.